CONTRIBUTING.md

```diff
@@ -127,6 +127,10 @@ That will use the version of `web-ext` installed by this project.
 As with the build instructions above, after running `yarn build:safari` you
 should be able to run using Xcode.
 
+Note that Xcode will default to signing with Brian Birtles' ([@birtles](https://github.com/birtles)) team. You may need to override the team name to your personal team—however, please don't commit the configuration files this will change to this repo. This is clunky, but unavoidable for open source projects with Xcode.
+
+Also, if you already have 10ten Japanese Reader installed on your device, you may get signing errors when trying to test the development version. An uninstall and reinstall should fix these.
```

Comment:
Do you recall having to set up Development Provisioning Profiles to deploy to your own device?

If so, worth providing this link:
https://developer.apple.com/help/account/manage-profiles/create-a-development-provisioning-profile/

And worth mentioning that one can develop for Simulator without a provisioning profile at all (I think – do you recall?).

Reply:
> Do you recall having to set up Development Provisioning Profiles to deploy to your own device?

I don't think so, or at least if I did, I can't remember it 😅.

> And worth mentioning that one can develop for Simulator without a provisioning profile at all (I think – do you recall?).

Hmm, I'm pretty I tried that and could not get it to work even on the simulator—however, that could easily just be my own ineptitude with Xcode. Will check this.

Reply:
Yep, it does work on a simulator without a profile set. Thanks for pointing that out! Will clarify the info.